smk-edit/static/components/edit-item.css

```diff
@@ -1,3 +1,6 @@
+.modal.edit-item .modal-content {
+    height: calc(100vh - 56px);
```

Comment:
Apparently this css isn't quite enough to solve all the issue with the UI. When I run the smk editor app, there is some UI that isn't visible. The horizontal line below metadata URL and the bottom arrow control of the scroll bar.

![image](https://user-images.githubusercontent.com/7094320/134091469-173b3429-9de3-47dd-b8a9-db69e2402556.png)

Reply:
I have another option that seems to work well and doesn't break anything that I can see. It also allows us to maintain the original order of the inputs (ie. no need to move the Data Url input. Note the v-bind:style="{position:'relative'}" attribute on the parent div of the checkbox in the following code snippet:
`
<div>
    <div class="row">
        <input-textarea class="col s12"
            v-model="dataUrl"
        >Data URL</input-textarea>
    </div>
    <div class="row">
        <input-select class="col s12"
            v-model="displayMode"
        >How should the data be displayed?
            <template v-slot:options>
                <option value="raw">Show features as styled</option>
                <option value="cluster">Show features in clusters</option>
                <option value="heatmap">Show features as a heatmap</option>
            </template>
        </input-select>
    </div>
    <div class="row" v-bind:style="{position:'relative'}">
        <input-checkbox v-if="displayMode === 'cluster'" class="col s12"
            v-model="showCoverageOnHover"
        >Show the bounds of a cluster on hover</input-checkbox>
    </div>
</div>
`

Reply:
Great suggestion! I've applied it and confirmed there's no longer a jump (even after restoring the original ordering of widgets) in Chrome, Firefox and Safari.